recipe_scrapers/marleyspoon.py

```diff
@@ -0,0 +1,146 @@
+import re
+
+from ._abstract import AbstractScraper
+from ._exceptions import RecipeScrapersExceptions
+from ._utils import get_minutes, normalize_string
+
+
+class UnsupportedLocale(RecipeScrapersExceptions):
+    """No support for selected locale of this website"""
+
+    def __init__(self, lang):
+        self.lang = lang
+        message = f'Selected locale "{self.lang}" is not supported.'
+        super().__init__(message)
+
+
+class MarleySpoon(AbstractScraper):
+    @classmethod
+    def host(cls, domain="de"):
+        return f"marleyspoon.{domain}"
+
+    def canonical_url(self):
+        return self.soup.find("meta", {"property": "og:url"})["content"]
+
+    def author(self):
+        return normalize_string(self.soup.find(class_="author-info").a.get_text())
+
+    def title(self):
+        return normalize_string(
+            self.soup.find("h3", {"class": "recipe-title"}).get_text()
+        )
+
+    def _specs(self):
+        vocab = {
+            "de": {
+                "time": "Dauer",
+                "category": "Schwierigkeitsgrad",
+                "nutrients": "Nährwertangaben",
+            },
+            "nl": {
+                "time": "Bereidingstijd",
+                "category": "Niveau",
+                "nutrients": "Voedingswaarde",
+            },
+            "en": {
+                "time": "Serving Time",
+                "category": "Level",
+                "nutrients": "Nutrition",
+            },
+        }
+
+        self.locale = self.language()[:2]
```

Comment:
Perhaps not strictly necessary in this case, but on the possibility that this pattern is copy-pasted to other scrapers:

```suggestion
        self.locale = self.language()[:2].lower()
```

The rationale there is to attempt to map some variance in the input to a key identifier format.